jax/experimental/checkify.py

```diff
@@ -212,6 +214,24 @@ def check_errors_traceable(msgs, err, code, *args):
   yield outs
 
 
+## assert primitive
+
+def assert_(pred: Bool, msg: str) -> None:
```

Comment:
nit: could probably use a nicer public symbol name, but we can think of that later? I guess you're not supposed to overwrite keywords, and we can't provide a `checkify.assert`?

Reply:
I think we can't overwrite `assert` actually; it's a syntax error. (I tried, of course!)

You're right that this name is bad. But what's a good one? `jassert`? `assert2`? `asssert`? `jax_assert`? `buttert`? (Brainstormed those with @skye...)

Reply:
`allege`?